draft-packages/tag/docs/Tag.stories.tsx

```diff
@@ -31,117 +23,136 @@ export default {
-    <Tag variant="validationNegative">Negative</Tag>
-    <Tag variant="validationCautionary">Cautionary</Tag>
-  </StoryContainer>
+const StickerSheetTemplate: Story<{ isReversed: boolean }> = ({
+  isReversed,
+}) => (
+  <>
+    <StoryWrapper isReversed={isReversed}>
+      <StoryWrapper.RowHeader
+        gridColumns={10}
+        headings={["Default", "Live", "Draft"]}
+      />
+      <StoryWrapper.Row gridColumns={10} rowTitle="Status (md)">
+        <Tag variant="default" size="medium">
+          Report
+        </Tag>
+        <Tag variant="statusLive" size="medium">
+          Live
+        </Tag>
+        <Tag variant="statusDraft" size="medium">
+          Draft
+        </Tag>
+      </StoryWrapper.Row>
+      <StoryWrapper.Row gridColumns={10} rowTitle="Status (sm)">
+        <Tag variant="default" size="small">
+          Report
+        </Tag>
+        <Tag variant="statusLive" size="small">
+          Live
+        </Tag>
+        <Tag variant="statusDraft" size="small">
+          Draft
+        </Tag>
+      </StoryWrapper.Row>
+    </StoryWrapper>
+    <StoryWrapper isReversed={isReversed}>
+      <StoryWrapper.RowHeader
+        gridColumns={10}
+        headings={["Neutral", "Postive", "Negative"]}
+      />
+      <StoryWrapper.Row gridColumns={10} rowTitle="Sentiment (md)">
+        <Tag variant="sentimentNeutral" size="medium">
+          Report
+        </Tag>
+        <Tag variant="sentimentPositive" size="medium">
+          Live
+        </Tag>
+        <Tag variant="sentimentNegative" size="medium">
+          Draft
+        </Tag>
+      </StoryWrapper.Row>
+      <StoryWrapper.Row gridColumns={10} rowTitle="Sentiment (sm)">
+        <Tag variant="sentimentNeutral" size="small">
+          Report
+        </Tag>
+        <Tag variant="sentimentPositive" size="small">
+          Live
+        </Tag>
+        <Tag variant="sentimentNegative" size="small">
+          Draft
+        </Tag>
+      </StoryWrapper.Row>
+    </StoryWrapper>
+    <StoryWrapper isReversed={isReversed}>
+      <StoryWrapper.RowHeader
+        headings={["Informative", "Postive", "Negative", "Cautionary"]}
+        gridColumns={7}
+      />
+      <StoryWrapper.Row gridColumns={7} rowTitle="Validation (md)">
+        <Tag variant="validationInformative" size="medium">
+          Informative
+        </Tag>
+        <Tag variant="validationPositive" size="medium">
+          Positive
+        </Tag>
+        <Tag variant="validationNegative" size="medium">
+          Negative
+        </Tag>
+        <Tag variant="validationCautionary" size="medium">
+          Cautionary
+        </Tag>
+      </StoryWrapper.Row>
+    </StoryWrapper>
+    <StoryWrapper isReversed={isReversed}>
+      <StoryWrapper.RowHeader gridColumns={10} headings={["Base"]} />
+      <StoryWrapper.Row gridColumns={10} rowTitle="Profile (md)">
+        <Tag
+          variant="profile"
+          avatar={
+            <Avatar
```

Comment:
I'd probably still include the example where `avatar={{}}` to show what we do as the fallback